tests/test_prealignment.py

```diff
@@ -0,0 +1,61 @@
+import urllib
+
+import numpy as np
+import PIL.Image as Image
+import pytest
+
+from tiatoolbox.tools.registration.prealignment import prealignment
+
+
+def test_prealignment():
```

Comment:
I cannot see a check to confirm that prealignment is correct. You can use some metrics.

Reply:
I will add a separate function for metrics that could be used to evaluate prealignment and DFBR output. There will be a separate PR for that.